Gems/Multiplayer/Code/Source/NetworkEntity/EntityReplication/EntityReplicator.cpp

```diff
@@ -33,6 +33,29 @@
 
 namespace Multiplayer
 {
+    void SyncActivatingEntityTransform(AZ::Entity* entity)
+    {
+        NetworkTransformComponent* netTransform = entity->FindComponent<NetworkTransformComponent>();
+        AzFramework::TransformComponent* transformComponent = entity->FindComponent<AzFramework::TransformComponent>();
```

Comment:
But this find can be removed.. TransformComponent is cached since it's so frequently used. You can get the TransformInterface directly off the entity itself and avoid the linear component search.

Reply:
Apparently this actually is needed. It's early enough that no TransformInterfaces are actually registered so the cache get fails. @SergeyAMZN might have a better understanding given he authored this piece.

Reply:
Correct, it's because we sync **Activating** entity transform. Transform Component is cached **after** the entity is activated.